Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -104,7 +104,17 @@ public extension String {
 		guard let first = self.first else { return nil }
 		return String(first)
 	}
-
+    
+    /// SwifterSwift: First character of string uppercased(if applicable) while keeping the original string.
+    ///
+    ///        "hello world".firstCharacterUppercased -> Optional("Hello world")
+    ///        "".firstCharacterUppercased -> nil
+    ///
+    public var firstCharacterUppercased: String? {
```

Comment:
Why not just use `capitalized`, or better yet `localizedCapitalized`?

Reply:
Because capitalize will uppercase first characters in every word, not practical when your localized strings are in the form of a sentence message

Reply:
not using localizedCapitalized in this code is simply because localizedCapitalized's deployment target is 9.0